File: online-bookstore-server/microservice-bookstore-user/src/main/java/com/onlinebookstore/entity/Address.java
package com.onlinebookstore.entity;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

/**
 * @author rkc
 * @version 1.0
 * @date 2020/9/11 9:35
 */
@Data
@ToString
@NoArgsConstructor
@AllArgsConstructor
public class Address {

    /*
    主键自增长字段
     */
    private Integer id;

    /*
    账号表的外键字段
     */
    @JsonProperty("account_username")
    private String accountUsername;

    /*
    地址
     */
    private String address;

    /*
    账户对象
     */
    private Account account;
}

File: online-bookstore-android/app/src/main/java/com/rkc/onlinebookstore/model/user/Address.kt
package com.rkc.onlinebookstore.model.user

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

/**
 * @author rkc
 * @date 2020/11/19 17:35
 * @version 1.0
 */
@Parcelize
data class Address(
    @SerializedName("id") var id: Int = -1,
    @SerializedName("account_username") var accountUsername: String = "",
    @SerializedName("phone") var phone: String = "",
    @SerializedName("receiver_name") var receiverName: String = "",
    @SerializedName("address") var address: String = ""
) : Parcelable